Jenkinsfile.k8s

```diff
@@ -112,18 +98,35 @@ pipeline {
         container('dealii'){
           sh '''
             export OMPI_MCA_btl=self,tcp
-            rm -f /home/dealii/build-gcc-fast/FAILED
-            cd /home/dealii/build-gcc-fast
-            ctest --output-on-failure -j4 || { touch FAILED; }
-            echo "Generating reference output..."
-            ninja generate_reference_output
-            cd $WORKSPACE
-            git checkout tests/CMakeLists.txt
+            cd /home/dealii/build-gcc-fast/tests
+            ninja -k 0 tests || true
```

Comment:
I have no idea what the rest of all of this does, but is this really what you want to do? It succeeds even if there are failures. Or are failures expected with the tests?